File: Part1/Day1/ifElse.java
import java.util.*;
public class ifElse {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int mark;
		
		System.out.print("Enter your mark: ");
		mark = sc.nextInt();
		sc.close();
		
		if(mark > 80)
			System.out.println("A");
		else if(mark > 60)
			System.out.println("B");
		else if(mark > 50)
			System.out.println("C");
		else if(mark > 45)
			System.out.println("D");
		else if(mark > 25)
			System.out.println("E");
		else
			System.out.println("F");
	}
}

File: Part1/Day1/ifElse.cpp
//#include "iostream"
#include "bits/stdc++.h"

using namespace std;

int main()
{
    // if-esle
    int mark;
    
    cout << "Enter your mark: ";
    cin >> mark;
    
    if(mark >= 80 && mark <= 100)
        cout << "A" << endl;
    else if(mark >= 60 && mark <= 79)
        cout << "B" << endl;
    else if(mark >= 50 && mark <= 59)
        cout << "C" << endl;
    else if(mark >= 45 && mark <= 49)
        cout << "D" << endl;
    else if(mark >=25 && mark <=44)
        cout << "E" << endl;
    else
        cout << "F" << endl;
        
    return 0;
}
